### ISSUE TITLE
[13] fleet: Missing vendor domain/default values

### ISSUE THREAD

**Issue by sswapnesh** [odoo/odoo#49744]
Impacted versions: 13.0 

Steps to reproduce:
![screenshot-localhost_8069-2020 04 18-22_19_08](https://user-images.githubusercontent.com/16624719/79643866-b5165680-81c2-11ea-90c2-b3e67eb2c5f7.png)

![screenshot-localhost_8069-2020 04 18-22_20_54](https://user-images.githubusercontent.com/16624719/79643890-e98a1280-81c2-11ea-83b9-e83f2d069ef4.png)

Current behavior: It is displaying all Contacts

Expected behavior: Should display only "Vendors"

Video/Screenshot link (optional):

**Comment by Yenthe666** [comment]
Just make a PR or OPW for these really small ones please :) this just fills up the issue counter with very minor things in my opinion. @pedrobaeza do you agree?

**Comment by pedrobaeza** [comment]
Yes, as you are able to perform the PR due to your skills, for sure it takes you the same time as opening the issue and it reduces the friction to solve these little things.

**Comment by sswapnesh** [comment]
I'm not sure if it will be merged or not. ( Just have a look at my opens PRs, Almost Each one is important but Still open since long)

**Comment by Yenthe666** [comment]
I totally get your point - and how it feels. The thing is that while PR's are not very actively managed issues are even worse. As you've probably noticed most of the Github issues are handled/managed/communicated on by me or @pedrobaeza and they have an even lower priority I'm afraid.
